collector/filesystem_freebsd.go

```diff
@@ -73,15 +73,16 @@ func (c *filesystemCollector) GetStats() (stats []filesystemStats, err error) {
 			ro = 1
 		}
 
-		labelValues := []string{device, mountpoint, fstype}
 		stats = append(stats, filesystemStats{
-			labelValues: labelValues,
-			size:        float64(fs.Blocks) * float64(fs.Bsize),
-			free:        float64(fs.Bfree) * float64(fs.Bsize),
-			avail:       float64(fs.Bavail) * float64(fs.Bsize),
-			files:       float64(fs.Files),
-			filesFree:   float64(fs.Ffree),
-			ro:          ro,
+			device:     device,
```

Comment:
Yes, I just realize that we don't run circleci for the pull requests which would have catched this (and me breaking master with my meminfo changes). Opened #401 for that.